src/inspector_socket.cc

```diff
@@ -559,21 +559,22 @@ static void init_handshake(InspectorSocket* inspector) {
   settings->on_url = path_cb;
 }
 
-int inspector_accept(uv_stream_t* server, InspectorSocket* inspector,
+// XXX(sam) shouldn't all these snake_case functions be CamelCase?
```

Comment:
Perhaps I misunderstand node naming conventions, but I thought C++ functions were supposed to be `CamelCase`, and only variables are supposed to be `snake_case`.